samples/04_gis_analysts_data_scientists/model_explainability_using_shap_for_tabular_data.ipynb

```diff
@@ -0,0 +1,534 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    import shap
</pre><p>Should we provide steps to install shap?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/977/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>SHAP will come with the conda metapackage and explicit installation would not be necessary.</p>

Reply:
<p>SHAP will come with the conda metapackage and explicit installation would not be necessary.</p>

Reply:
<p>sounds good</p>